packages/sharing-smart-contract/test/subscription.test.js

```diff
@@ -215,4 +269,60 @@ describe('Subscription.sol', () => {
       ).to.be.revertedWith('ProtectedData is not in collection');
     });
   });
+
+  describe('removeProtectedDataFromSubscription()', () => {
```

Comment:
add test case
- cannot removeProtectedDataFromSubscription if the collection has ongoing subscriptions